frontend/ui/quickstart.lua

```diff
@@ -10,61 +10,185 @@ local lfs = require("libs/libkoreader-lfs")
 local _ = require("gettext")
 
 local QuickStart = {
-    quickstart_force_show_version = 201511982,
+    quickstart_force_show_version = 2021070000,
```

Comment:
Not sure what this value should be :/

Reply:
I'd go with YYYYMMDD like the one-time-update stuff ;).

Reply:
Gah, except the current one is larger.

Reply:
So, YYYYMMDD0 ? :D

Reply:
I think I did try a 9 digit YYYYMMDD0 , but had to go with 10 digits as the thing we compare with is 10 digits. (version stuff has probably changed since 2015 :)
Just wondering about current release 202107 vs upcoming nightlies vs next release 202108 - the thing we store and compare to is not even the previous value there, but may be the version at install time? Bit confusing.

Reply:
Oh, right, it's ultimately compared to `Version:getNormalizedCurrentVersion()`, so, YYYYMMRRRR, where YYYYMM is indeed the latest stable's date, and RRRR the amount of commits since (so, you can probably use 0000 right now, meaning `2021070000`).

Reply:
Which is exactly what you did, I'm blind xD.